packages/bs-loader/index.js

```diff
@@ -17,10 +17,10 @@ try {
 
 const bsb =
   os.platform() === 'darwin'
-    ? `script -q /dev/null ${bsbCommand} -make-world -color`
+    ? `script -q /dev/null ${bsbCommand} -clean-world -make-world -color`
```

Comment:
I added `-clean-world` here because warnings don't show when running `bsb` if there are no changes since the last compile

Reply:
Is there anyway to display warnings without "-clean-world"? It's a pretty costly operation, I don't know how it would affect build performance